stubs/keyboard/keyboard/__init__.pyi

```diff
@@ -0,0 +1,169 @@
+
+# is_pressed cannot check multi-step hotkeys, so not using _ParseableHotkey
+
+def is_pressed(hotkey: _Key | _ScanCodeList) -> bool: ...
+def call_later(fn: Callable[..., None], args: _P = ..., delay: float = ...) -> None: ...
+def hook(callback: _Callback, suppress: bool = ..., on_remove: Callable[[], None] = ...) -> Callable[[], None]: ...
+def on_press(callback: _Callback, suppress: bool = ...) -> Callable[[], None]: ...
+def on_release(callback: _Callback, suppress: bool = ...) -> Callable[[], None]: ...
+def hook_key(key: _ParseableHotkey, callback: _Callback, suppress: bool = ...) -> Callable[[], None]: ...
+def on_press_key(key: _ParseableHotkey, callback: _Callback, suppress: bool = ...) -> Callable[[], None]: ...
+def on_release_key(key: _ParseableHotkey, callback: _Callback, suppress: bool = ...) -> Callable[[], None]: ...
+def unhook(remove: _Callback) -> None: ...
+
+unhook_key = unhook
+
+def unhook_all() -> None: ...
+def block_key(key: _ParseableHotkey) -> Callable[[], None]: ...
+
+unblock_key = unhook_key
+
+def remap_key(src: _ParseableHotkey, dst: _ParseableHotkey) -> Callable[[], None]: ...
+
+unremap_key = unhook_key
+
+def parse_hotkey_combinations(hotkey: _ParseableHotkey) -> tuple[tuple[tuple[int, ...], ...], ...]: ...
+def add_hotkey(
+    hotkey: _ParseableHotkey,
+    callback: Callable[..., bool | None],
+    args: _P = ...,
+    suppress: bool = ...,
+    timeout: float = ...,
+    trigger_on_release: bool = ...,
+) -> Callable[[], None]: ...
+
+register_hotkey = add_hotkey
+
+def remove_hotkey(hotkey_or_callback: _ParseableHotkey | _Callback) -> None: ...
+
+unregister_hotkey = remove_hotkey
+clear_hotkey = remove_hotkey
+
+def unhook_all_hotkeys() -> None: ...
+
+unregister_all_hotkeys = unhook_all_hotkeys
+remove_all_hotkeys = unhook_all_hotkeys
+clear_all_hotkeys = unhook_all_hotkeys
+
+def remap_hotkey(
+    src: _ParseableHotkey, dst: _ParseableHotkey, suppress: bool = ..., trigger_on_release: bool = ...
+) -> Callable[[], None]: ...
+
+unremap_hotkey = remove_hotkey
+
+def stash_state() -> list[int]: ...
+def restore_state(scan_codes: Iterable[int]) -> None: ...
+def restore_modifiers(scan_codes: Iterable[int]) -> None: ...
+def write(text: str, delay: float = ..., restore_state_after: bool = ..., exact: bool | None = ...) -> None: ...
+def wait(hotkey: _ParseableHotkey | None = ..., suppress: bool = ..., trigger_on_release: bool = ...) -> None: ...
+def get_hotkey_name(names: Iterable[str] | None = ...) -> str: ...
+def read_event(suppress: bool = ...) -> KeyboardEvent: ...
+def read_key(suppress: bool = ...) -> _Key: ...
+def read_hotkey(suppress: bool = ...) -> str: ...
+def get_typed_strings(events: Iterable[KeyboardEvent], allow_backspace: bool = ...) -> Generator[str, None, None]: ...
+def start_recording(
+    recorded_events_queue: Queue[KeyboardEvent] | None = ...,
+) -> tuple[Queue[KeyboardEvent], Callable[[], None]]: ...
+def stop_recording() -> list[deque[KeyboardEvent]]: ...
```

Comment:
I think it's just a list of KeyboardEvents.

Reply:
You're right, I got confused into thinking it was a list of deque, but it's just converting